### ISSUE TITLE
Add dropdown field

### ISSUE THREAD

**Issue by braindead1** [Zefau/ioBroker.jarvis#20]
My lawn mower has the following modes: Auto (0), Manual (1), Home (2) and End of Day (3). Currently it is not possible to show the mode in Jarvis and switch between the modes.

Please add the possibility to select different options. The most simple way could be a dropdown field.

**Comment by Zefau** [comment]
It is possible to show the modes (instead of the value) via `jarvis`, see https://github.com/Zefau/ioBroker.jarvis/wiki/Ger%C3%A4te-(Devices)#feld-jarvis.

Example:
```
{
   "state":{
      0:"Auto",
      1:"Manual",
      2:"Home",
      3:"End of Day"
   }
}
```

Though, it is not possible (at the moment) to trigger a change.

**Comment by Zefau** [comment]
- [ ] add the possibility so set an individual display name for each device
- [ ] add the possibility so set individual action triggers (e.g. Button, Switch, Dropdown or whatever) for each device

**Comment by braindead1** [comment]
One comment, because I think there may be a misunderstanding: It should be possible to set the display name of a device per widget.

If you have a tab for your Esszimmer and a tab for your blinds, the device "Esszimmer Rolladen" could be named "Rolladen" on the Esszimmer tab and "Esszimmer" on the blinds tab.

**Comment by Zefau** [comment]
I labelled this issue with `Widget StateList`. For that one I see the need to have a display name for a device that differs from the actual name. 
Not all widgets have devices though, like `AdapterStatus` or `DateTime`. Or it is not necessary (from my perspective) like in `Map`.

So the need for that is indeed widget-specific and necessary for `StateList`.

**Comment by Zefau** [comment]
`v1.0.0-beta.1` is available on Github! This introduces a very detailed configuration of the `StateList` module and introduces a `DropdownAction`:

![image](https://user-images.githubusercontent.com/14368811/91656595-49ba8a80-eaba-11ea-92c3-dab74983e8a4.png)

`Show states` will show the selected states, such as:
![image](https://user-images.githubusercontent.com/14368811/91656615-71a9ee00-eaba-11ea-874b-d0105020f24d.png)
_(example only with primary state)_

`Show an action element` allows to trigger an action via a specific component, such as:
![image](https://user-images.githubusercontent.com/14368811/91656631-91411680-eaba-11ea-9b1f-c0210279c3ba.png)
_(example is a `SwitchAction`)_

or

![image](https://user-images.githubusercontent.com/14368811/91656634-9c944200-eaba-11ea-85a0-9b6670bcb100.png)
_(example is a `DropdownAction`)_

____

Furthermore, the `Body` can be configured as well, such as:
![image](https://user-images.githubusercontent.com/14368811/91656659-d36a5800-eaba-11ea-83fe-e2d1d0d6573d.png)
_(example is a `LevelBody`)_

**Comment by Zefau** [comment]
@braindead1 Looking forward for your feedback! Thanks in advanced.

**Comment by braindead1** [comment]
Can you please explain in detail how to use `DropdownAction`? I was trying it for the last 20 minutes without success. My lawn mower has the following modes: `{
      "0": "Auto",
      "1": "Manuell",
      "2": "Home",
      "98": "End of day",
      "99": "Job"
    }`

**Comment by Zefau** [comment]
In the device configuration define the state of the device and paste `{ "0": "Auto", "1": "Manuell", "2": "Home", "98": "End of day", "99": "Job" }` into the display field. Then select that State in combination with the DropdownAction.

**Comment by Zefau** [comment]
> Besides that LevelBody is not working for me, no matter what combination of "Body State Key" and "Level Body" is configure.

Is the selected state of the number? Do you get any errors?

**Comment by braindead1** [comment]
> In the device configuration define the state of the device and paste `{ "0": "Auto", "1": "Manuell", "2": "Home", "98": "End of day", "99": "Job" }` into the display field. Then select that State in combination with the DropdownAction.

That's exactly what I tried. The screen stays gray, jarvis is not loaded and the following error appears in console
![image](https://user-images.githubusercontent.com/2136249/91663814-8357b980-eaeb-11ea-8dae-8dff5a424ea4.png)

**Comment by Zefau** [comment]
Could you try if this one has been fixed with `1.0.0-beta.2`?

**Comment by braindead1** [comment]
Still not working. There are not errors in the console.

![image](https://user-images.githubusercontent.com/2136249/91669092-81edb780-eb12-11ea-9635-8a40c9503354.png)

This is my configuration:

![image](https://user-images.githubusercontent.com/2136249/91669076-566acd00-eb12-11ea-8e8e-0d66d60f34fb.png)

![image](https://user-images.githubusercontent.com/2136249/91669082-6e425100-eb12-11ea-89d3-0bcece1e8d54.png)

**Comment by Zefau** [comment]
Looks absolutely correct. Could you share the JSON part of both the device and the layout configuration?

**Comment by braindead1** [comment]
Since v1.0.0-beta.1 jarvis is not working anymore for me. At the moment there are a lot of bugs, for most of them there is either an issue or they are described in one of my comments. I attached my devices and the layout.

[layout.txt](https://github.com/Zefau/ioBroker.jarvis/files/5164566/layout.txt)
[devices.txt](https://github.com/Zefau/ioBroker.jarvis/files/5164567/devices.txt)

**Comment by Zefau** [comment]
`1.0.0-beta.4` should solve this issue.

**Comment by braindead1** [comment]
I can confirm that this issue is solved in `v1.0.0-beta.4`.

**Comment by Zefau** [comment]
21d0842fd27f7ae6c1e71879b026b00fbfd70487
